WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeOrDiscountLineDetails.swift

```diff
@@ -30,12 +30,12 @@ struct FeeLineDetails: View {
                 VStack(spacing: .zero) {
                     Section {
                         if viewModel.isPercentageOptionAvailable {
-                            Picker("", selection: $viewModel.feeType) {
-                                Text(viewModel.percentSymbol).tag(FeeLineDetailsViewModel.FeeType.percentage)
-                                Text(viewModel.currencySymbol).tag(FeeLineDetailsViewModel.FeeType.fixed)
+                            Picker("", selection: $viewModel.feeOrDiscountType) {
```

Comment:
When tapping in the Picker between `%` and `$` I get these errors in the console:

<img width="408" alt="Screenshot 2023-07-12 at 07 13 25" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/fb07a87c-8f41-4168-a7ff-4b36385761cd">

```
2023-07-12 07:11:45.984681+0200 WooCommerce[45114:5211027] Error: string input is not a number:
2023-07-12 07:11:52.142560+0200 WooCommerce[45114:5211027] Error: string input is not a number:
```